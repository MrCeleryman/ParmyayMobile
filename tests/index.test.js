import "react-native";
import React from "react";
import renderer from "react-test-renderer";
import Index from "../src/index";

describe("app", () => {
	it("renders correctly", () => {
		const tree = renderer.create(
			<Index />
		);
	});

	it("Snapshot matches", () => {
		expect(renderer.create(
			<Index />
		)).toMatchSnapshot();
	});
});

