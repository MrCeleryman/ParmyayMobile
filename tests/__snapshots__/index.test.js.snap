// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app Snapshot matches 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#F5FCFF",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "fontSize": 20,
        "margin": 10,
        "textAlign": "center",
      }
    }
  >
    Welcome to Parmyaymobile
  </Text>
</View>
`;
